assets/large_language_models/rag/environments/rag_embeddings/context/conda_dependencies.yaml

```diff
@@ -41,11 +42,13 @@ dependencies:
   - pandas>=1
   - polling2~=0.5.0
   - psutil~=5.8.0
+  - pymongo # explicitly install for [azure_cosmos_mongo_vcore, azure_cosmos_nosql]
```

Comment:
I think pymongo install for [azure_cosmos_mongo_vcore, mongodb], azure_cosmos_nosql doesn't use pymongo

Reply:
You are right. I will update the comment.